omnibus/config/software/jmxfetch.rb

```diff
@@ -33,9 +33,9 @@
     delete "jmxfetch.jar"
 
     if ENV['HARDENED_RUNTIME_MAC'] == 'true'
-      hardened_runtime = '-o runtime '
+      hardened_runtime = "-o runtime --entitlements #{entitlements_file} "
```

Comment:
We are being extra safe here, because probably Java is strict enough with its memory management to not need `allow-unsigned-executable-memory`